src/main/java/frc/robot/commands/AimAtTarget.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Navigation;
+import frc.robot.subsystems.drive.Drive;
+
+public class AimAtTarget extends Command {
+    Drive driveSubsystem;
+    Navigation navSubsystem;
+    int targetId;
+
+    public AimAtTarget(Drive driveSubsystem, Navigation navSubsystem, int targetId) {
+        this.driveSubsystem = driveSubsystem;
+        this.navSubsystem = navSubsystem;
+        this.targetId = targetId;
+
+        addRequirements(driveSubsystem, navSubsystem);
+    }
+
+    @Override
+    public void initialize() {
+        navSubsystem.setDesiredTarget(targetId);
```

Comment:
Yes, I think it's the pose of the target, but all targets should be centered above the speaker, etc. so shouldn't the angle still work?

Reply:
If we have passed in the speaker's AprilTag target ID based on our alliance color that could work

Reply:
Yes that is what I was working on setting up in constants, so one could say constants.speakertag and it would be set to current field alliance

Reply:
Ideally the alliance color would be received from the DriverStation (there's a way to request the current alliance color) rather than being a hardcoded constant. This would need to run in RobotContainer similar to what Charlie has done in the main repo

Reply:
ok ill update that now